source/ogre/common/TerrainBlend.cpp

```diff
@@ -124,7 +126,7 @@ void CScene::UpdBlendmap()
 	pt->lock(HardwareBuffer::HBL_DISCARD);
 
 	const PixelBox& pb = pt->getCurrentLock();
-	float* pD = static_cast<float*>(pb.data);
+	Ogre::uchar* pD = static_cast<Ogre::uchar*>(pb.data);
```

Comment:
This one slipped in this unrelated commit -- GCC 8 was throwing an error that a `static_cast` from `Ogre::uchar* const` to `float*` was not possible, so I did this change. Not sure if it's a good one or if it can cause issues.